infra/modules/providers/azure/ml-workspace/terraform.tfvars

```diff
@@ -0,0 +1,6 @@
+resource_group_name     = ""
```

Comment:
Is it possible to rename to `terraform.tfvars.template` to make it clear that this is actually a template of what the vars might look like?